src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -3619,19 +3619,27 @@ static void writeBeam(XmlWriter& xml, ChordRest* const cr, Beam* const b)
         const auto crst = elements[i];
         if (crst->isChord()) {
             blp = toChord(crst)->beams();
+        } else if (crst->isRest()) {
+            blp = toRest(crst)->beams();
         }
     }
-    // find beam level current chord
+    // find beam level current chord or rest
     if (cr->isChord()) {
         blc = toChord(cr)->beams();
         bmc = toChord(cr)->beamMode();
+    } else if (cr->isRest()) {
+        blc = toRest(cr)->beams();
+        bmc = toRest(cr)->beamMode();
     }
     // find beam level next chord
```

Comment:
... or rest

Reply:
There's another one, furter up. Actually 4 more

Also wouldn't a simple `else` be sufficient? What else can a `ChordRest` be but a `Chord` or a `Rest`?